### ISSUE TITLE
Center the chart in a specific zoom and xaxis value

### ISSUE THREAD

**Issue by juanmafer** [apexcharts/apexcharts.js#955]
Hi!,
I use a stacked bar chart that is updated via ajax.
According we need, the ajax return one bar per day, week, month or year.
I need to center the results and set the zoom according the "time range" utilized.

For example if I select one bar per week, I need to set the zoom to view just the last 4 bars (the last month). And if I choice one bar per day, I need to set the zoom to view the last 30 bars (the last month).
It's posible?
THKS!

**Comment by junedchhipa** [comment]
You have to calculate the min and max of xaxis on your side, and then use the `updateOptions` function to redraw the chart.

```javascript
xaxis: {
  min: timestamp,
  max: timestamp,
}
```

illustrated in this example - https://apexcharts.com/javascript-chart-demos/area-charts/datetime-x-axis/
